File: updraft-sdk/src/main/java/com/appswithlove/updraft/api/response/FeedbackEnabledResponse.java
package com.appswithlove.updraft.api.response;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class FeedbackEnabledResponse {

    @SerializedName("is_feedback_enabled")
    private Boolean mIsFeedbackEnabled;

    @SerializedName("error_code")
    private List<String> mErrorCodes;

    @SerializedName("error_description")
    private List<String> mErrorDescriptions;

    public Boolean getFeedbackEnabled() {
        return mIsFeedbackEnabled;
    }

    public void setFeedbackEnabled(Boolean feedbackEnabled) {
        mIsFeedbackEnabled = feedbackEnabled;
    }

    public List<String> getErrorCodes() {
        return mErrorCodes;
    }

    public void setErrorCodes(List<String> errorCodes) {
        mErrorCodes = errorCodes;
    }

    public List<String> getErrorDescriptions() {
        return mErrorDescriptions;
    }

    public void setErrorDescriptions(List<String> errorDescriptions) {
        mErrorDescriptions = errorDescriptions;
    }
}

File: updraft-sdk/src/main/java/com/appswithlove/updraft/api/response/FeedbackEnabledResponse.kt
package com.appswithlove.updraft.api.response

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class FeedbackEnabledResponse(
    @SerialName("is_feedback_enabled") val isFeedbackEnabled: Boolean,
    @SerialName("error_code") val errorCodes: List<String> = emptyList(),
    @SerialName("error_description") val errorDescriptions: List<String> = emptyList(),
)
